src/widgets/chart.rs

```diff
@@ -8,18 +9,37 @@ use crate::{
     layout::{Alignment, Constraint, Rect},
     style::{Color, Style, Styled},
     symbols,
-    text::{Line as TextLine, Span},
+    text::{Line, Span},
     widgets::{
-        canvas::{Canvas, Line, Points},
+        canvas::{Canvas, Line as CanvasLine, Points},
```

Comment:
Renamed the canvas line instead of the text line because otherwise that would show up on the doc as `TextLine`

Reply:
I've been generally using the approach of using `text::Line` instead of the rename - it's close enough to the same amount of code. No big deal though.

Reply:
That's a good alternative, I'll remember that. Let's keep it that way since its already done?